extensions/objects/templates/FilepathEditor.js

```diff
@@ -165,13 +171,28 @@ oppia.directive('filepathEditor', [
           });
         };
 
-        $scope.filepathsLoaded = false;
-        $http.get(
-          '/createhandler/resource_list/' + $scope.explorationId
-        ).then(function(response) {
-          $scope.filepaths = response.data.filepaths;
-          $scope.filepathsLoaded = true;
-        });
+        $scope.generateImageFilename = function() {
+          var format = '';
+          var chunks = $scope.state.uploadedFile.name.split('.');
```

Comment:
I need to store it anyway? Because this function is only called later on when the user decides to "save" the image.